org.codeberg.dnkl.foot.yml

```
@@ -0,0 +1,67 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
```

```suggestion
runtime-version: "22.08"
```

it's out now. might as well.

---

The used Sdk extension didn't provide this version currently

---

as I said, it's out now, might as well.

I know it wasn't available before.

---

I mean `org.freedesktop.Sdk.Extension.golang` not the Freedesktop Sdk.

---

the goland sdk is now build for 22.08 (will be available in a few hours)